src/Internal.AspNetCore.SiteExtension.Sdk/build/Internal.AspNetCore.SiteExtension.Sdk.targets

```diff
@@ -17,8 +17,8 @@
     </PropertyGroup>
 
     <ItemGroup Condition="'@(HostingStartupRuntimeStoreFrameworks->Count())' == '0'">
-      <HostingStartupRuntimeStoreTargets Include="netcoreapp2.1" Runtime="win7-x64" />
```

Comment:
Maybe we should make this `$(TargetFramework)` so it updates automatically with the project. Thoughts @pakrym ?

Reply:
SiteExtension itself targets net461 because it never mattered:
https://github.com/aspnet/AzureIntegration/blob/dev/extensions/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj#L6

I'm fine with changing it to target framework here but we need to fix consumption side too.

Reply:
By comsumption side you mean in the csproj in AzureIntegration? What would happen if we did the change here but not there?

Reply:
It would be failing the same way it does now.